crypto/fipsmodule/rand/rand.c

```diff
@@ -48,9 +48,22 @@
 // This might be a bit of a leap of faith, esp on Windows, but there's nothing
 // that we can do about it.)
 
+// When in FIPS mode we use the CPU Jitter entropy source to seed our DRBG.  
+// This entropy source is very slow and can incur a cost anywhere between 10-60ms
+// depending on configuration and CPU.  Increasing to 2^24 will amortize the 
+// penalty over more requests.  The value was selected by reviewing OpenSSL and 
+// SP 800-90B for the max reseed interval (2^48)
+//
+// CPU Jitter:  https://www.chronox.de/jent/doc/CPU-Jitter-NPTRNG.html
+// 
 // kReseedInterval is the number of generate calls made to CTR-DRBG before
 // reseeding.
+
+#if defined(BORINGSSL_FIPS)
```

Comment:
Nit: we should try and use `AWSLC_FIPS` in the future. But we will change this in bulk at a later stage.